cluster/src/main/java/org/apache/iotdb/cluster/metadata/CMManager.java

```diff
@@ -416,6 +424,35 @@ public void createSchema(PhysicalPlan plan) throws MetadataException {
     }
   }
 
+  private void verifyCreatedSgSuccess(List<PartialPath> deviceIds) throws MetadataException {
+    long startTime = System.currentTimeMillis();
+    while (true) {
+      int successNum = deviceIds.size();
+      for (PartialPath deviceId : deviceIds) {
+        PartialPath storageGroupName = MetaUtils
+          .getStorageGroupPathByLevel(deviceId, IoTDBDescriptor.getInstance()
+            .getConfig().getDefaultStorageGroupLevel());
+        if (IoTDB.metaManager.isStorageGroup(storageGroupName)) {
+          successNum --;
+        }
+      }
+      if (successNum <= 0) {
+        break;
+      }
+
+      if (System.currentTimeMillis() - startTime > ClusterDescriptor.getInstance().getConfig().getConnectionTimeoutInMS()) {
+        break;
+      } else {
+        try {
+          Thread.sleep(1);
+        } catch (InterruptedException e) {
+          logger.debug("Failed to wait for creating sgs for plan");
```

Comment:
It would be better if  the log here contains specified log~

Reply:
ok